Makefile

```diff
@@ -21,3 +21,6 @@ lint-master:
 	$(git_diff_to_master)
 	npx --yes ts-standard --stdin --stdin-filename DIFF
 
+test:
+	make dev
+	docker exec -it paybutton-dev sh -c "CI=true yarn test"
```

Comment:
I think this command does not work:
![image](https://user-images.githubusercontent.com/21281174/172246217-5cde2879-d9a6-47f9-80e2-7d591933b03a.png)

Reply:
Removed the `it` argument as it was running interactively, checking if action now goes through.